packages/babel-plugin-proposal-object-rest-spread/README.md

```diff
@@ -54,18 +54,28 @@ require("@babel/core").transform("code", {
 
 ## Options
 
+By default, this plugin will produce spec compliant code. The Babel's `objectSpread` helper will be used.
+
+### `loose`
+
+`boolean`, defaults to `false`.
+
+Enabling this option will use Babel's `extends` helper.
```

Comment:
Would be could if we could add a note that `extends` is `Object.assign` + a fallback and explain a difference between using assign and defineProperty as some sort of caveat paragraph about `loose` mode in this transform.

Reply:
Mhh okay! Will try to add it later as soon as I'll have some free time 🙂

Reply:
Actually there is already a link about that dfference somewhere at the bottom, so maybe it's just a matter of linking to it from here too.